src/main/java/maria/Parser.java

```diff
@@ -0,0 +1,115 @@
+package maria;
+
+import java.time.LocalDate;
+
+import maria.command.Command;
+import maria.command.CommandAddDeadline;
+import maria.command.CommandAddEvent;
+import maria.command.CommandAddTodo;
+import maria.command.CommandExit;
+import maria.command.CommandFindTask;
+import maria.command.CommandHelp;
+import maria.command.CommandListAllTasks;
+import maria.command.CommandMarkTask;
+import maria.command.CommandRemoveTask;
+import maria.command.CommandUnknown;
+import maria.command.CommandUnmarkTask;
+
+/**
+ * Parses commands.
```

Comment:
Not sure if it is required, but perhaps the JavaDoc comment could be more descriptive?